docs/build-on-polkadot/xcm/asset-transfer-api/reference.md

```diff
@@ -0,0 +1,168 @@
+
+# Asset Transfer API Reference 
+# ( 🚧 WIP )
+
+<br>
+<div class="grid cards" markdown>
+-   :octicons-download-16:{ .lg .middle } __Install the Asset Transfer API__
+
+    ---
+
+    Learn how to install [`asset-transfer-api`](#) into a new or existing project.
+
+    <br>
+    [:octicons-arrow-right-24: Get started](#)
+
+-   :octicons-code-16:{ .lg .middle } __Dive in with a tutorial__
+
+    ---
+
+    Ready to start coding? Follow along with a step-by-step tutorial.
+
+    <br>
+    [:octicons-arrow-right-24: How to use the Asset Transfer API](#)
+</div>
+<br>
+
+
+## Asset Transfer API Class
+
+Holds open an API connection to a specified chain within the ApiPromise in order to help construct transactions for assets and estimate fees.
+
+### Methods
+
+#### createTransferTransaction
+
+This function allows you to create XCM transactions to transfer assets or native tokens between different chains.
+
+It takes several parameters to specify the transfer details, including the destination chain, recipient address, assets to be transferred, and their amounts, as well as an optional parameter for further customization. It can infer what kind of transaction is necessary given the inputs. When sending cross-chain transfers, the API performs extensive validation to ensure the inputs are valid and the assets either exist or don't.
+
+!!! note
+	The `createTransferTransaction` function is designed to be a utility that simplifies the creation of the transaction. It does not sign or submit the created transaction on the blockchain. It simply generates the transaction in the requested format (e.g., payload, call, or submittable). After obtaining the transaction from the `createTransferTransaction` function, you will need to handle the signing and submission process separately.
+
+```ts
+public async createTransferTransaction<T extends Format>(
+  destChainId: string,
+  destAddr: string,
+  assetIds: string[],
+  amounts: string[],
+  opts: TransferArgsOpts<T> = {},
+): Promise<TxResult<T>>
+```
+
+??? interface "Request parameters"
+
+	`destChainId` ++"string"++ <span class="required" markdown>++"required"++</span>
+	
+	ID of the destination chain ('0' for Relay chain, other values for parachains)
+
+	---
+
+	`destAddr` ++"string"++ <span class="required" markdown>++"required"++</span>
+
+	Address of the recipient account on the destination chain
+
+	---
+
+	`assetIds` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+	Array of asset IDs to be transferred
+      
+    If a token or asset ID is provided as input, the API will resolve to using the `tokens` pallet. When no asset is passed in, the API will default to using the `balances` pallet.
+
+	---
+
+	`amounts` ++"string[]"++ <span class="required" markdown>++"required"++</span>
+
+	Array of amounts corresponding to each asset in `assetIds`
+
+	---
+
+	`opts` ++"TransferArgsOpts<T>"++
+
+    --8<-- 'code/build-on-polkadot/xcm/asset-transfer-api/reference/transfer-arg-opts.md'
```

Comment:
This is not an excuse, but my mind was between this is a list, and this is not, so that's why there are places with periods and places where I did not use them. I think I'll keep them in this format, I also like it